README.md

```diff
@@ -29,6 +29,12 @@ The code is available in the [python package index](https://pypi.org/project/agn
 pip install agnpy
 ```
 
+The code can also be installed using `conda`
+
+```bash
+conda install -c conda-forge agnpy
```

Comment:
If you prefer, could use the `astropy` channel here instead. It is the same underlying package in either case.

Reply:
It's fine as is